tests/test_models.py

```diff
@@ -2015,7 +2014,7 @@ def test_fork_callback(self):
     @mock.patch('website.archiver.tasks.archive')
     def test_register_callback(self, mock_archive):
         registration = self.node.register_node(
-            None, self.consolidate_auth, '', '',
+            None, self.consolidate_auth, {}
```

Comment:
Use `AuthUserFactory` in the setup method for `self.user`, and replace all usages of `consolidate_auth` with `self.user.auth`.